test/apiv2/10-images.at

```diff
@@ -186,6 +186,11 @@ BUILD_TEST_ERROR=""
 if ! grep -q '200 OK' "${TMPD}/headers.txt"; then
     echo -e "${red}NOK: Image build from tar failed response was not 200 OK (application/x-tar)"
     BUILD_TEST_ERROR="1"
+
+    if ! grep -qP '^{"aux":{"ID":"sha256:[0-9a-f]{64}"}}$' "${TPMD}/response.txt"; then
```

Comment:
Typo: should be `TMPD`. I have no idea how this is passing CI, but this is a huge red flag that your test is not doing what you think it's doing.

Reply:
It passed through sheer dumb luck. I fixed the test to actually test the change, and immediately found a bug. It should work now.